src/sentry/seer/anomaly_detection/utils.py

```diff
@@ -30,7 +35,36 @@ def translate_direction(direction: int) -> str:
     return direction_map[AlertRuleThresholdType(direction)]
 
 
-NUM_DAYS = 28
+def get_snuba_query_string(snuba_query: SnubaQuery) -> str:
+    """
+    Generate a query string that matches what events-stats does
```

Comment:
not sure what event-stats are from this comment, might be useful to add more context to this comment block (or omit it) - since it's kinda the same as the method name.

Reply:
clarified with the endpoint name